turbine_models/tools/power_curve_tools.py

```diff
@@ -0,0 +1,163 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+def plot_power_curve(wind_speed, cp_curve, ct_curve):
+    """Plot Cp and Ct curve per wind speed.
+
+    Args:
+        wind_speed (list | np.ndarray): list of wind speeds in m/s
+        cp_curve (list | np.ndarray): power curve coefficients (Cp) at each wind speed in ``wind_speed``
+        ct_curve (list | np.ndarray): thrust curve coefficients (Ct) at each wind speed in ``wind_speed``
+    """
+    fig1 = plt.figure()
+    plt.plot(wind_speed, ct_curve, label="Coeff of Thrust")
+    plt.plot(wind_speed, cp_curve, label="Coeff of Power")
+    plt.legend()
+    plt.xlabel("Wind Speed [m/s]")
+    plt.show()
```

Comment:
```suggestion
    fig = plt.figure()
    ax = fig.add_subplot(111)

    ax.plot(wind_speed, ct_curve, label="Coeff of Thrust")
    ax.plot(wind_speed, cp_curve, label="Coeff of Power")
    ax.legend()
    ax.set_xlabel("Wind Speed [m/s]")
    fig.show()
```
Using the `plt.figure()` method of initialization should use the `add_suplot()` logic for plots like the suggested. That or drop the `plt.figure()`, though I'm wildly biased in favor of the suggested, separate figure and axes object approach.